package.json

```diff
@@ -37,15 +37,19 @@
     "seed:prod": "NODE_ENV=production lerna run seed --stream --scope @opencrvs/data-seeder",
     "add:license": "license-check-and-add add -f license-config.json",
     "build:components": "lerna run build --scope @opencrvs/components",
-    "debug": "bash debug-service-in-chrome.sh"
+    "debug": "bash debug-service-in-chrome.sh",
+    "knip": "knip"
```

Comment:
Do we need the command here? Is it useful and commonly used enough to be added to the basic commands?